File: service/CustomerService.cs
using LabMPP.domain;
using LabMPP.domain.Validators;
using LabMPP.repository;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LabMPP.service
{
    class CustomerService
    {
        private CustomerRepo customerRepo;
        private Validator<Customer> validator = new CustomerValidator();

        public CustomerService(CustomerRepo customerRepo)
        {
            this.customerRepo = customerRepo;
        }

        public Customer getCustomer(long idUser)
        {
            return customerRepo.getOne(idUser);
        }
    }
}

File: Services/src/main/java/service/CustomerService.java
package service;

import domain.Customer;
import domain.validators.CustomerValidator;
import domain.validators.Validator;
import repository.CustomerRepo;

import java.io.Serializable;

public class CustomerService implements Serializable {
    private CustomerRepo customerRepo;
    private Validator<Customer> validator=new CustomerValidator();
    public CustomerService(CustomerRepo customerRepo) {
        this.customerRepo = customerRepo;
    }

    public Customer getCustomer(Long idUser) {
        return customerRepo.getOne(idUser);
    }
}
